configs/google_com_http3_upstream_proxy.yaml

```diff
@@ -0,0 +1,62 @@
+admin:
+  address:
+    socket_address:
+      protocol: TCP
+      address: 0.0.0.0
+      port_value: 9901
+static_resources:
+  listeners:
+  - name: listener_0
+    address:
+      socket_address:
+        protocol: TCP
```

Comment:
UDP?

Reply:
I think the idea here is a standard TCP listener that makes a h3 upstream call to h3 enabled www.google.com

Reply:
Oh, my bad, thinking it was downstream.

Reply:
oh I hoped the naming would be enough - let me add a comment at the top of the configs too!